### ISSUE TITLE
Bug - A11y issues flagged by updated pf-a11y tool

### ISSUE THREAD

**Issue by nicolethoen** [patternfly/patternfly#7193]
**Describe the problem**
This is an issue with 
- [ ] Patternfly 5  
- [x] Patternfly 6  
- [ ] other

When attempting to update to the latest version of the patternfly-a11y dependency (5.0.0) new a11y issues are getting flagged
See a11y report from the [dependabot's version bump PR](https://github.com/patternfly/patternfly/pull/7191): https://patternfly-pr-7191-a11y.surge.sh/

- [x] card demo has invalid `aria-controls` values (the `aria-controls` on a table's expandable row toggle does not match the id of the expandable content) (fixed by https://github.com/patternfly/patternfly/pull/7194)
- [ ] dividers in a list cannot have a `role="separator"` - I suggest using aria-hidden in those cases?
- [ ] drawers which use a draggable splitter are missing a required `aria-valuenow` attribute. This is because axe assumes that an element with `role="separator"` and `tabindex=0` is being used as an adjustable component and so it needs an `aria-valuenow` prop. In the case of the core docs, since the JS has not enabled actual adjusting of the splitters, adding the attribute `aria-valuenow="50"` will resolve this issue
- [x] multiple table examples have invalid `aria-controls` values (the `aria-controls` on a table's expandable row toggle does not match the id of the expandable content) (fixed by https://github.com/patternfly/patternfly/pull/7194)
- [ ] multiple table examples have a `toggle all` control in their `thead` which has an invalid `aria-controls` value. In such cases, the `aria-controls` value should be a space delimited list of the ids of each row's expandable sections.
- [ ] Tabs (`ul` elements with `role='tablist'`) cannot have children with any role other than `role='tab'`. So all of our examples which include a button beside the tab for additional actions, such as information icons or close buttons, are violating that rule. This may require some redesign?

Once these are all addressed, we could consider turning back on the CI rule that requires the a11y tool to pass in order to merge the PR

**Comment by mcoker** [comment]
There is an open PR that shows all of the a11y failures that need to be addressed - https://github.com/patternfly/patternfly/pull/7352

**Comment by thatblindgeye** [comment]
FWIW the errors regarding Tabs and required aria children, the temporary fix in React was to include `'aria-required-children'` in the a11y config file ignore array so we could do the same in Core for now. Long term we should definitely look into re-evaluating not only this implementation in Tabs, but similar implementations with Menus with favorites/actions next to the main menu item. We have a React issue related to the topic: https://github.com/patternfly/patternfly-react/issues/9968
